_includes/example-verifiable-presentation.html

```diff
@@ -0,0 +1,161 @@
+      ],
+      "type": [
+        "VerifiableCredential",
+        "AlumniCredential"
+      ],
+      "issuer": {
+        "id": "did:key:z6MkfMMx7yTbRBpjTL5PyiHjucZoyjZ3vEXod3d3Yvn2znez",
+        "image": "data:image/png;base64,iVBORw0KG....QYFB0T/BzE/ye22Qk+VAAAAAElFTkSuQmCC"
+      },
+      "name": "Alumni Credential",
+      "description": "The holder is an alumni of Example University.",
+      "issuanceDate": "2023-08-14T15:16:21.585Z",
+      "credentialSubject": {
+        "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+        "alumniOf": {
+          "identifier": "did:example:c276e12ec21ebfeb1f712ebc6f1",
+          "name": "Example University"
+        }
+      },
+      "id": "https://playground.chapi.io/credential/gOCqUCjewfMeE4SojdCaj",
+      "proof": {
+        "type": "Ed25519Signature2020",
+        "created": "2023-08-14T15:16:21Z",
+        "verificationMethod": "did:key:z6MkfMMx7yTbRBpjTL5PyiHjucZoyjZ3vEXod3d3Yvn2znez#z6MkfMMx7yTbRBpjTL5PyiHjucZoyjZ3vEXod3d3Yvn2znez",
+        "proofPurpose": "assertionMethod",
+        "proofValue": "z8FM4tCT6LYf8NjFBP1tV1S6zp4XcpNUXsWzpKZ7KTPSAPbW9RVxh2wEJ6acdqL1YyQqnCNbNAUVsjxDZmauoDUb"
+      }
+    },
+    {
+      "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/citizenship/v1",
+        "https://w3id.org/security/suites/ed25519-2020/v1"
+      ],
+      "id": "https://issuer.oidp.uscis.gov/credentials/83627465",
+      "type": [
+        "VerifiableCredential",
+        "PermanentResidentCard"
+      ],
+      "issuer": {
+        "id": "did:key:z6MkjEPj5Qb3AwG8VBQHcALVVYba7pfTPYQUX5tDs7ujJrTa",
+        "image": "data:image/png;base64,iVBORw0KG....AAAAASUVORK5CYII="
+      },
+      "identifier": "83627465",
+      "name": "Permanent Resident Card",
+      "description": "Government of Utopia Permanent Resident Card.",
+      "issuanceDate": "2023-08-07T00:30:06.886Z",
+      "expirationDate": "2029-12-03T12:19:52Z",
+      "credentialSubject": {
+        "id": "did:example:b34ca6cd37bbf23",
+        "type": [
+          "PermanentResident",
+          "Person"
+        ],
+        "givenName": "JANE",
+        "familyName": "SMITH",
+        "gender": "Female",
+        "image": "data:image/png;base64,iVBORw0KG....Q9eh9EeIAAAAABJRU5ErkJggg==",
+        "residentSince": "2015-01-01",
+        "lprCategory": "C09",
+        "lprNumber": "999-999-999",
+        "commuterClassification": "C1",
+        "birthCountry": "Arcadia",
+        "birthDate": "1978-07-17"
+      },
+      "proof": {
+        "type": "Ed25519Signature2020",
+        "created": "2023-08-07T00:30:06Z",
+        "verificationMethod": "did:key:z6MkjEPj5Qb3AwG8VBQHcALVVYba7pfTPYQUX5tDs7ujJrTa#z6MkjEPj5Qb3AwG8VBQHcALVVYba7pfTPYQUX5tDs7ujJrTa",
+        "proofPurpose": "assertionMethod",
+        "proofValue": "z5VrEHiQUCL2VWzZX7HfBbR8Ruf6SaoAFJUbygVJv8wpsx12PnR3EN4ZC925uU1nNa7yFWsFEkVPdc859aBVC86EX"
+      }
+    },
+    {
```

Comment:
We should use the JFF PF3 VC.

Reply:
Happy to switch from the Alumni one. Noted.